Automattic/wp-calypso
# Odie: cleanup Wapuu checks

## Proposed Changes

Move all logic that determines if Wapuu/Odie should be shown into a single hook.

1. Checking the flags twice [1](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-chat-status.ts#L46), [2](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-still-need-help-url.tsx#L19)
2. [This function should not be exported this file](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-still-need-help-url.tsx#L11-L15)
3. [This check](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-chat-status.ts#L46) is not needed as all users should be receiving Wapuu if they are allowed to have support.

## Why are these changes being made?

Currently there are a few places where this is being checked and a few of them are redundant. This will simplify the logic and make it easier to understand.

## Testing Instructions

This needs to be checked against free and paid users. If you are a free user you should receive contact form for FORUMS otherwise you should see Wapuu when you click on the button `Still need help?`

1. test from a free user and verify Help Center support works as decribed above
4. test from a paid user and verify you get Wapuu.
